packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue

```diff
@@ -6,15 +6,17 @@
         <span>{{ distance }}km</span> away from you
       </div>
     </slot>
-    <div class="sf-store__media" @click="centerOn(latlng)">
+    <div v-bind:class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
```

Comment:
```suggestion
    <div :class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
```

Wdyt about using shorthand?

Reply:
@Michal-Dziedzinski Thanks for the valid suggestion. I'll update the PR according to it.